tutorials/02-example.py

```diff
@@ -7,6 +7,450 @@
+# speed to apply, so let's pull some online data first.
+#
+# We are going to retrieve some data from NASA regarding the wind speed at
+# Ansys Madrid office in Paseo de la castellana.
+#
+# .. figure:: ../images/office1.jpg
+#     :width: 600px
+#     :align: center
+#     :alt: Madrid office
+#     :figclass: align-center
+#
+#     Paseo de la castella Ansys office (Madrid).
+#     Yes, we love coffee.
+#
+#
+#
+# From: https://power.larc.nasa.gov/data-access-viewer/
+#
+
+import json
+
+import pandas as pd
+import requests
+
+latitude, longitude = (40.447488, -3.691763)
+parameters = ["T2M_MAX", "T2M_MIN"]
+
+base_url = r"https://power.larc.nasa.gov/api/temporal/daily/point?parameters={parameters}&community=RE&longitude={longitude}&latitude={latitude}&start=20200101&end=20210305&format=JSON"
+api_request_url = base_url.format(
+    longitude=longitude, latitude=latitude, parameters=",".join(parameters)
+)  # Another way to format f-strings!
+
+response = requests.get(url=api_request_url, verify=True, timeout=30.00)
+
+content = json.loads(response.content.decode("utf-8"))
+df = pd.DataFrame(content["properties"]["parameter"])
+
+df.columns = ["MAX", "MIN"]  # renaming columns
+df = df.set_index(
+    pd.to_datetime(df.index, format="%Y%m%d")
+)  # Formatting dataframe index as date.
+
+#######################################################
+# Let's see the data
+df.head()
+
+######################################################
+# and describe it...
+df.describe()
+######################################################
+# We see there are negative wind speed, probably because of the direction,
+# since we are not interested in direction, only magnitude,
+# let's use the absolute value then.
+#
+df = df.abs()
+
+######################################################
+# Let's plot it...
+_ = df.plot(title="Wind speed per day")
+
+########################################################
+# As we can see, the most frequent maximum speeds are:
+_ = df["MAX"].hist(bins=20)
+
+########################################################
+# As we can see, we could stablish there are two main peaks,
+# one at 15 m/s and another at 33 m/s.
+#
+# Let's generate some random wind signal with those speeds.
+# We are going to do a superposition of harmonics:
+#
+
+amplitude = [15, 33]
+frequencies = (
+    np.array([10, 16]) * 2 * np.pi
+)  # Typical wind frequencies range between 2 and 20 Hz
+phase = np.random.random(size=len(frequencies)) * 2 * np.pi
+
+
+def wind_speed(t):
+    sum_ = 0
+    for each_amp, each_w, each_phase in zip(amplitude, frequencies, phase):
+        sum_ = sum_ + each_amp * np.cos(each_w * t + each_phase)
+    return sum_
+
+
+t = np.arange(0, 1, 0.01)
+
+plt.plot(t, wind_speed(t))
+plt.show()
+
+
+#############################################################
+# To apply these velocities, we are going instead to convert it to acceleration
+# using the following equation:
```

Comment:
```suggestion
# Instead of converting it to acceleration, we are going to use
# the following equation to apply these velocities:
```

Reply:
We got the typical wind velocities in paseo de la Castellana, with that we are going to create an harmonic signal for that velocity. Now, to apply it in the FE model, it is easier to apply accelerations using ACEL, so we convert those velocities to accelerations by multiplying it by its frequency. Does it make sense?

Reply:
Yes! Perfect sense. 

Maybe it's because I am not native, but the sentence sounds weird to me. I would say something like "To apply these velocities, we are going to convert them to accelerations using the following equation". I just got confused with the "instead to". But again, it may be grammatically correct, I don't really know.

Reply:
I think you are right.... I will change it tomorrow!